frontend/src/scenes/surveys/constants.tsx

```diff
@@ -146,3 +146,103 @@ export const NEW_SURVEY: NewSurvey = {
     archived: false,
     appearance: defaultSurveyAppearance,
 }
+
+export enum SurveyTemplateTypes {
+    Interview = 'User interview',
+    NPS = 'Net promoter score (NPS)',
+    CSAT = 'Customer satisfaction score (CSAT)',
+    CES = 'Customer effort score (CES)',
+    CCR = 'Customer churn rate (CCR)',
+    Superhuman = 'Product-market fit (Superhuman)',
+}
+
+export const defaultSurveyTemplates = [
+    {
+        type: SurveyTemplateTypes.Interview,
+        questions: [
+            {
+                type: SurveyQuestionType.Link,
+                question: 'Would you be interested in participating in a customer interview?',
+                description: 'We are looking for feedback on our product and would love to hear from you!',
+                link: 'https://calendly.com/',
+            },
+        ],
+        description: <>Send users straight to your calendar.</>,
+    },
+    {
+        type: SurveyTemplateTypes.NPS,
+        questions: [
+            {
+                type: SurveyQuestionType.Rating,
+                question: 'How likely are you to recommend us to a friend?',
+                description: '',
+                display: 'number',
+                scale: 10,
+                lowerBoundLabel: 'Unlikely',
+                upperBoundLabel: 'Very likely',
+            },
+        ],
+        description: 'Get an industry-recognized benchmark.',
+    },
+    {
+        type: SurveyTemplateTypes.Superhuman,
+        questions: [
+            {
+                type: SurveyQuestionType.SingleChoice,
+                question: 'How would you feel if you could no longer use PostHog?',
+                choices: ['Not disappointed', 'Somewhat disappointed', 'Very disappointed'],
+            },
+        ],
+        description: "If 40% say 'very disappointed', you're at product-market fit.",
```

Comment:
```suggestion
        description: "40% 'very disappointed' signals product-market fit.",
```

Just shortening this to keep it on one line